packages.bzl

```diff
@@ -82,6 +82,7 @@ MATERIAL_SCSS_LIBS = [
 
 GOOGLE_MAPS_PACKAGES = [
     "google-map",
+    "google-map-marker",
```

Comment:
I thought about this for awhile and I think you're right, that it should be in the same module. However, if that's the case, then we also shouldn't have a separate module for google-map. In other words, we should just have '@angular/google-maps' instead of '@angular/google-maps/google-map'. That's a pretty big refactor though, do you think it should be done in the same pull request?

Reply:
That will be fine in a follow-up. I never actually realized it wasn't one main module for the whole thing.

Reply:
Okay, that sounds good. For this one, I will connect google-map-marker to the google-map module, and in the subsequent cl, I will have both of them be in a google-maps module.

Reply:
I ran into some unexpected problems having google-map remain a separate subpackage but integrating map-marker. So, to avoid having to double my work, I decided to just combine everything into one package in this pr. It makes this already very large pr even bigger, but it should correct the build failures you saw.